src/d3a/constants.py

```diff
@@ -45,3 +45,5 @@
 CN_PROFILE_EXPANSION_DAYS = 7
 
 RUN_IN_REALTIME = False
+
+FEED_IN_TARIFF = 20
```

Comment:
We want to change this only in a blue moon or why is this not a GlobalConfig parameter? Not an expert here, but doesn't this price also change frequently?

Reply:
> We want to change this only in a blue moon or why is this not a GlobalConfig parameter?

Looks better there, will move it even now

> Not an expert here, but doesn't this price also change frequently?

I haven't changed it yet as for all three referenced cases, feed-in tariff had the same rate.